creating-a-project.md

```diff
@@ -1,5 +1,27 @@
 ## Creating a Project
 
+First, make sure you've installed jspm. You can install jspm globally:
+
+```
+npm install --global jspm@0.17-beta.14
+```
+
+Or install it locally to your project:
+
+```
+npm install --save-dev jspm@0.17-beta.14
+```
+
+_Once jspm 0.17 is stable and released you'll be able to just run `npm install jspm`, but for the beta version you must specify the version._
+
+Note that if you install it locally the `jspm` executable will be placed within `./node_modules/.bin` and you'll have to execute it like so:
+
+```
+./node_modules/.bin/jspm
```

Comment:
This isn't really advisable, and is always used by the global jspm anyway. So the best workflow is actually to install *both* globally and locally.